docs/Block-Node-Types.md

```diff
@@ -0,0 +1,126 @@
+# Block Node Types
+
+## Overview
+
+Block Nodes can be a somewhat slippery concept. The core system encompasses a
```

Comment:
Should we drop the first sentence?
I fell like it won't be applicable much longer especially with efforts like this but I get you're highlighting that it's a weighted term. 
```suggestion
Block Nodes  encompass a
```